### PR TITLE
[bitnami/mariadb]: Add support for RuntimeClass

### PR DESCRIPTION
**Pull request by RamyAllam**
### Description of the change

Adding support for RuntimeClass into MariaDB helm chart

### Benefits

It allows the MariaDB workload to run on FireCracker, CloudHypervisor, Sysbox, gVisor...etc

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mariadb/templates/primary/statefulset.yaml [hunk 1]
```diff
@@ -72,6 +72,11 @@ spec:
       {{- else if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName | quote }}
       {{- end }}
+      {{- if .Values.primary.runtimeClassName }}
+      runtimeClassName: {{ .Values.primary.runtimeClassName | quote }}
+      {{- else if .Values.runtimeClassName }}
```
Comment (migruiz4) [inline bitnami/mariadb/templates/primary/statefulset.yaml:77]: The value `.Values.runtimeClassName` has not been added to the `values.yaml` file.
Please add the missing value or replace the logic to only use primary/secondary values.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
The PR references the value `runtimeClassName` in the root of the values.yaml, but it hasn't been declared, could you please take a look at my comment?

File: bitnami/mariadb/templates/primary/statefulset.yaml (hunk 1 above)
Comment (RamyAllam) [reply bitnami/mariadb/templates/primary/statefulset.yaml:77]: It's added, Thanks 👍

File: bitnami/mariadb/Chart.yaml [hunk 2]
```diff
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mariadb
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 11.3.5
+version: 11.3.6
```
Comment (migruiz4) [inline bitnami/mariadb/Chart.yaml:29]: As this is a new feature, could you please bump a new minor version `11.4.0`?

**Review by migruiz4 — CHANGES_REQUESTED** [review]
One final request before merging, please bump a minor version (new feature) instead of a patch (fix)

File: bitnami/mariadb/Chart.yaml (hunk 2 above)
Comment (RamyAllam) [reply bitnami/mariadb/Chart.yaml]: Sure, it's done.

**Review by migruiz4 — APPROVED** [review]
Thank you very much for applying the suggestions so quickly! Looks good to me!

**Pull request merged**